Python/Product/Profiling/PythonProfilingPackage.cs

```diff
@@ -287,12 +287,17 @@ protected override int CreateToolWindow(ref Guid toolWindowType, int id)
         private static void ProfileStandaloneTarget(SessionNode session, StandaloneTarget runTarget, bool openReport) {
             LaunchConfiguration config;
             if (runTarget.PythonInterpreter != null) {
-                var registry = session._serviceProvider.GetComponentModel().GetService<IInterpreterRegistryService>();
-                var interpreter = registry.FindConfiguration(runTarget.PythonInterpreter.Id);
-                if (interpreter == null) {
-                    return;
+                if (runTarget.PythonInterpreter.Id == Strings.LaunchProfiling_OtherInterpreter) {
```

Comment:
@int19h I don't think I can use `is CustomPythonInterpreterView` here to distinguish custom vs regular interpreters. The `runTarget` is a `StandaloneTarget` object, and the `runTarget.PythonInterpreter` is a `PythonInterpreter` project. So I have to compare `id` here. Do you see better options? :)